<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataBrowserWidget</class>
 <widget class="QWidget" name="dataBrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Flowsheet Results</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="menuButton">
          <property name="text">
           <string>Menu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Current Filter:</string>
          </property>
          <property name="buddy">
           <cstring>filterSelectBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filterSelectBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editFiltersButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Add/Edit Filters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="columnsButton">
          <property name="text">
           <string>Columns</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveEnsembleButton">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Save as New Ensemble</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Number of Rows:</string>
          </property>
          <property name="buddy">
           <cstring>numRowsBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numRowsBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="AlternateBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>220</red>
              <green>255</green>
              <blue>220</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="AlternateBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>220</red>
              <green>255</green>
              <blue>220</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="AlternateBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>220</red>
              <green>255</green>
              <blue>220</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
